### PR TITLE
fix: add deprecation warning to ci bash scripts

### PR DESCRIPTION
**Pull request by hugorut**
* adds deprecation overview to ci README
* adds warning comment in scripts
* adds warning log to scripts

### CODE REVIEW

File: scripts/ci/jenkins_diff.sh [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+# This script is DEPRECATED and is no longer maintained.
+#
+# Please migrate to our new Jenkins integration: https://github.com/infracost/infracost-jenkins
+#
+# This script will be removed September 2022.
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+echo "Warning: this script is deprecated and will be soon removed."
+echo "Please visit https://github.com/infracost/infracost/blob/master/scripts/ci/README.md for instructions on how to upgrade."
```
Comment (alikhajeh1) [inline scripts/ci/jenkins_diff.sh:13]:
```suggestion
echo "Warning: this script is deprecated and will be removed in Sep 2022."
echo "Please visit https://github.com/infracost/infracost-jenkins/ for the new recommended way of using Infracost in Jenkins."
```

**Comment by hugorut** [comment]
@alikhajeh1 do we want to get this in the next release? Looking like tomorrow? Or hold off?

**Comment by alikhajeh1** [comment]
Thanks @hugorut - up to you - seems like it's ready and low-risk so we can ship it, but if it needs more testing then it can wait.

**Comment by hugorut** [comment]
@aliscott I'm going to merge this in for the release today - any pushback on that?

**Comment by aliscott** [comment]
What's the reasoning for Sep 2022?

**Comment by hugorut** [comment]
@aliscott I believe that's what was settled on in the kanban discussion task. I'm pro removing sooner though.

**Comment by mloskot** [comment]
The time has passed :)

https://github.com/infracost/infracost/blob/e3cc239ea07ab7a974a01158ab9c7e2c00a96be7/scripts/ci/comment.sh#L10-L11

**Pull request merged**